docs/github.md

```diff
@@ -67,8 +67,10 @@ changes.
 ## Working in a Git co-located repository
 
 After doing `jj git init --colocate`, Git will be in a [detached HEAD
-state][detached], which is unusual, as Git mainly works with bookmarks. In a
-co-located repository, every `jj` command will automatically synchronize
+state][detached], which is unusual; this is because jj mainly works with
+bookmarks, not branches.
```

Comment:
comment: I don't know if this is emphasizing the right thing. I would say that this is unusual because jj mainly works with commits (or heads) rather than because it mainly works with bookmarks, so a "detached HEAD" state is unusual in Git but common in jj. You could also say it's unusual because Git strongly encourages that all work be reachable from some branch, while jj does not, or somehow express that.

Reply:
As suggested on Discord, among other things, I have left out mentioning bookmarks altogether. That also goes along with your other comment about the intended meaning of the original sentence likely being that Git uses (named) branches, not jj using bookmarks.